### PR TITLE
Patch VTX link and fix all remaining links

### PR DESCRIPTION
**Pull request by haslinghuis**
This should be the final link change on configurator :slightly_smiling_face:

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]
* approving, but i saw two items without space before `target` (non-breaking).

**Pull request merged**
